src/App.tsx

```diff
@@ -1,10 +1,50 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodosList } from './TodosList';
+import { InputField } from './InputField';
+import { ErrorMessage } from './handleError';
+import { Footer } from './Footer';
 
-const USER_ID = 0;
+type TodoStatus = 'all' | 'active' | 'completed';
+
+const USER_ID = 10627;
 
 export const App: React.FC = () => {
+  const [todosList, setTodosList] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<TodoStatus>('all');
+  const [itemsLeft, setItemsLeft] = useState(0);
+
+  const [isError, setIsError] = useState(false);
```

Comment:
You can set the error message to a string instead
```suggestion
  const [isError, setIsError] = useState('');
```